packages/lib-classifier/src/store/SubjectGroup/SubjectGroup.js

```diff
@@ -0,0 +1,77 @@
+import { getRoot, tryReference, types } from 'mobx-state-tree'
+import Resource from '../Resource'
+import subjectViewers from '../../helpers/subjectViewers'
+import SingleImageSubject from '../SingleImageSubject'
+
+function validateSubjects (subjects) {
+  return subjects.length > 1
+}
+
+const SubjectGroup = types
+  .model('SubjectGroup', {
+    already_seen: types.optional(types.boolean, false),
+    favorite: types.optional(types.boolean, false),
+    finished_workflow: types.optional(types.boolean, false),
+    metadata: types.frozen({}),
+    retired: types.optional(types.boolean, false),
+    selected_at: types.maybe(types.string),
+    selection_state: types.maybe(types.string),
+    shouldDiscuss: types.maybe(types.frozen()),
+    subjects: types.refinement(types.array(SingleImageSubject), validateSubjects),
+    user_has_finished_workflow: types.optional(types.boolean, false),
+  })
+
+  .views(self => ({
+    get locations () {
+      return self.subjects.map(subject => subject.locations[0])
+    },
+
+    get talkURL () {
+      const project = tryReference(() => getRoot(self).projects?.active)
+      if (project) {
+        const { origin } = window.location
+        return `${origin}/projects/${project.slug}/talk/subjects/${self.id}`
+      }
+
+      return ''
+    },
+
+    get viewer () {
+      return subjectViewers.subjectGroup
+    },
```

Comment:
[note to self]

I'm making a note of this since it'll probably minorly affect #1555

I'm still curious if the "selection logic" (in this case, determining which SubjectViewer to use) is best served by placing it in individual Subject sub-types' `get viewer()` code, and overriding the generic Subject type's `get viewer()` "selection crossroads".

I mean, pro: it _definitely_ results in more compartmentalised code that fits with the monorepo philosophy, but con: it _might_ make the selection logic harder to grok if the logic is split across several places.

(My feelings for/against this is exactly at the 50% mark, so I'm quite ambivalent.)

Reply:
Agreed with @shaunanoordin here. Having logic in different places for which viewer to render is harder to follow. Is there a reason why this isn't following the same logic as other customized viewers (LCV, VSLCV) in using a workflow configuration setting to select which subject viewer to render?

Reply:
I thought about copying over the `viewer` function from the `Subject` model. We can still do that.

Reply:
If we did copy over the `viewer` logic from `Subject`, then `groupSupject.viewer` could return viewer types other than the `GroupSubject` viewer. I'm not sure that's what we want.